content/post/a-place-to-start-gdd.md

```diff
@@ -0,0 +1,69 @@
+We gave ourselves a rough list of steps to start with, some points that we thought would be essential to understand the big picure of the game, using this question as a guideline. So far is working quite well.
+
+**Walking in your target's shoes helps providing valuable information and valuable information is what you need to craft a great product.**
+
+Fourth: a GDD it is a good starting point for the future. Let's say you decide to take your project to the next step, and find a company to develop it; the GDD - even though it is clearly not the tool to _sell_ your idea - is, in my opinion, a great tool to describe your game and it highlights the fact you have actually designed it, and took it seriously as a project.
+
+**A good design talks about your love for your project. A GDD can shout it out loud.**
+
+
+## What we are writing in our GDD
+
+When we started writing the document we also tried to create a list of what we thought it should have contained. A list helps us to undestand how much work still has to be done and, while working on a step, to note down anything relevant that could be useful in the next ones.
+
+This is the table of contents we made.
+Right now some of these points are far from done (not even started, actually) but that's part of the plan: we write down what we think it will be useful and, as soon as we get there, we decide if it still is. We'll try to update you about this in some future posts.
+
+* a seed, the abstract of the game plot
+* a brief description of the plot itself divided in four acts
+* a small section of aestethic influences
+ * what styles of graphics we like for a GA
+ * what styles we think are suitable for our GA
+* a schematic representation of the puzzles of the first act
+* a detailed description of the elements in the first act
+ * locations
+ * characters
+ * objects and items
+ * relevant lines of dialogue
+* a map of the connections between the locations
+
+## Some final thoughts about the GDD
+
+So far, this approach to our project seems to be a good one.
+I personally have some doubts about the nature of the GDD, that I will try to analyze with time. I always think that a methodology should be followed and trusted, but also adapted to the teams and situations, so I'm summarizing the biggest concerns about the GDD here, for future personal references and to share them with anyone interested.
+
+**Agility**
+Coming from the ICT we know that writing a big upfront document goes against those philosophies that would like the design and the development to be paired in a more agile strategy. I wonder if this applies to this context too.
+
+**Readability**
+We also know that a document is a linear artifact - it is something you read from the top to the bottom - and it's inadequate to represent something so complex as a game like a GA. For example, there are many objects and cross references between them.
+
+That's it :)
```

Comment:
I'd add a closing sentence about the next article. Since it is something I'll do, my proposal is:

```
That's it for the big picture :)
In the next post we'll talk about what will start the whole creative process: the adventure seed.
```